requirements.txt

```diff
@@ -26,3 +26,4 @@ PyYAML==3.12
 cryptography==2.2.2
 service-identity==17.0.0
 git+git://github.com/shwethanidd/pika.git@gevent_connection_adapter#egg=pika
+wget
```

Comment:
I think we need to provide a version number for wget as well. The latest is 3.2.

Reply:
Its not mandatory. It grabs the latest and I do only the basic download without any additional parameters. But added it anyway